test/failures_testsuite/testcases/test04_minio_active_passive.py

```diff
@@ -0,0 +1,169 @@
+from base_test import BaseTest
+import requests
+from requests.exceptions import ConnectionError
+import time
+from jumpscale import j
+import unittest
+
+
+class TestActivePassive(BaseTest):
+
+    def setUp(self):
+        if not self.s3_active_service_name:
+           self.skipTest('No s3 service for active minio is found')
```

Comment:
where did u import skipTest?
I'm not sure if it is working in `setUp` or not

Reply:
working